cfpb/cfgov-refresh
# Update Spanish homepage, homebuying page, and complaint process page

Implements Spanish website content updates per requests in https://[GHE]/design-ops/CEE-projects/issues/294

## Changes

- Relocates featured links from homepage hero to right-hand column below hero
- Renames/relocates homebuying page from "Hogar" to "Comprar Casa"
- Updates copy on complaint process page to mirror the current English version
- Tweaks a few styles

## Additions

- Links to "Cómo preparar sus finanzas antes de comprar una casa" and "El Dinero Mientras Creces" on the homepage
- Link to "Cómo preparar sus finanzas antes de comprar una casa" on the homebuying page

## Testing

1. Pull branch
1. `gulp styles`
1. `./runserver.sh`
1. Load http://localhost:8000/es/ and see the updated homepage
1. Click the "Comprar Casa" tab and see the updated homebuying page
1. Click the "Presentar una queja" and see the updated complaint process page (most notably, the removal of step 6 in the process)

## Screenshots

Homepage:
![screen shot 2017-12-01 at 18 31 02](https://user-images.githubusercontent.com/1044670/33681486-b86416da-da92-11e7-8467-279c19ae95e1.png)

Homebuying page:
![screen shot 2017-12-01 at 18 30 33](https://user-images.githubusercontent.com/1044670/33681501-c2a8ca00-da92-11e7-8b31-2569f265ca61.png)

Complaint process page:
![screen shot 2017-12-01 at 18 32 00](https://user-images.githubusercontent.com/1044670/33681508-ca69aade-da92-11e7-9959-2e588d63519e.png)

## Checklist
